### PR TITLE
Added Cartoonify Image Code

### PR DESCRIPTION
**Pull request by sharur7**
### Cartoonifying an image
The code converts your original image i.e. the image which you have given as input, to cartoonified image.
When you run Cartoonify.py, both the input image and output image will be displayed.

### HackinCodes Participant
This PR is associated with the issue:
#93 

@disha2sinha @ankitdobhal Please review my PR

### CODE REVIEW

File: Image-Processing/Cartoonify Image/Cartoonify.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Importing Libraries
+import cv2
+import numpy as np
+from skimage import io
+
+
```
Comment (disha2sinha) [inline Image-Processing/Cartoonify Image/Cartoonify.py]: Remove this new line

File: Image-Processing/Cartoonify Image/Cartoonify.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        cartoon = cv2.bitwise_and(color, color, mask=edges)
+        cartoon_img = cv2.cvtColor(cartoon, cv2.COLOR_BGR2RGB)
+
+        # Re-sizeing Image
+        resize = cv2.resize(RGB_img, (600, 450))
+        resize2 = cv2.resize(cartoon_img, (600, 450))
+        self.resize = resize
+        self.resize2 = resize2
+
+
```
Comment (disha2sinha) [inline Image-Processing/Cartoonify Image/Cartoonify.py]: Remove this new line

File: Image-Processing/Cartoonify Image/Cartoonify.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+        gray = cv2.medianBlur(gray, 5)
+        edges = cv2.adaptiveThreshold(gray, 255, cv2.ADAPTIVE_THRESH_GAUSSIAN_C, cv2.THRESH_BINARY, 9, 7)
+
+        # 2) Color Image
+        color = cv2.bilateralFilter(img, 10, 300, 300)
+        RGB_img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+
+        # 3) Cartoon Image
+        cartoon = cv2.bitwise_and(color, color, mask=edges)
```
Comment (disha2sinha) [inline Image-Processing/Cartoonify Image/Cartoonify.py:20]: give space around =

**Review by disha2sinha — CHANGES_REQUESTED** [review]

File: Image-Processing/Cartoonify Image/Cartoonify.py (hunk 1 above)
Comment (sharur7) [reply Image-Processing/Cartoonify Image/Cartoonify.py]: done

File: Image-Processing/Cartoonify Image/Cartoonify.py (hunk 2 above)
Comment (sharur7) [reply Image-Processing/Cartoonify Image/Cartoonify.py]: done

File: Image-Processing/Cartoonify Image/Cartoonify.py (hunk 3 above)
Comment (sharur7) [reply Image-Processing/Cartoonify Image/Cartoonify.py:20]: done

**Review by disha2sinha — APPROVED** [review]

**Comment by sharur7** [comment]
@ankitdobhal Please kindly merge this request

**Pull request merged**